invoice/src/main/java/org/killbill/billing/invoice/dao/InvoiceDao.java

```diff
@@ -149,4 +149,32 @@ InvoicePaymentModelDao createRefund(UUID paymentId, BigDecimal amount, boolean i
      * @throws InvoiceApiException
      */
     void changeInvoiceStatus(UUID invoiceId, InvoiceStatus newState, InternalCallContext context) throws InvoiceApiException;
+
+    /**
+     * Save parent/child invoice relationship
+     *
+     * @param invoiceRelation the invoice relation object
+     * @param context the tenant context
+     * @throws InvoiceApiException
```

Comment:
jdk8 generates lots of error if Javadocs are incomplete (missing description for `throws` here, similar to https://github.com/killbill/killbill-plugin-api/commit/194447ca1c223af00ad96ea41ab016ce7e8ac514).

Some background: http://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html.